Domains/1-Common/DistributionSystems/DistributionSystems.ecschema.xml

```diff
@@ -62,4 +62,83 @@
             </IsMixin>
         </ECCustomAttributes>
     </ECEntityClass>
+
+    <ECEnumeration typeName="DirectionType" backingTypeName="int" isStrict="true">

```

Comment:
Perhaps just `Direction` instead of `DirectionType`?

Reply:
Or `DistributionPortDirection`, `PortDirection`?

Reply:
we seem to have established a standard of suffixing our enums with "Type":
SectionType / AutoHandledPropertyStatementType /CustomHandledPropertyStatementType 

i suggest we follow it here.

Reply:
in the linked PR we seem to have settled on "Value" suffix? is that the new standard proposed?

my point was about us having some history publishing enums in biscore schema, so I suggest we either suffix it with Type or Value, otherwise we're making a mess...

Reply:
Grepping all schemas, the only place that seems to have established a clear standard is CIF, where they all end with "_Enum". In BisCore, 3 of 4 end in Type, but that is because they are semantically trying to convey the name of a type or kind of something... not just a potential value and the ECProperty they are paired with ends in "Type". Imagine an enum of standard allowable diameters... it is not exactly a type.
The 4th in BisCore ends in "Role".

I think what BisCore and most authors have done is just name the enum to match the property where it is used.

I agree with Karolis that reserving "Type" to refer to TypeDefinitionElement is more consistent, so if we were going to put an arbitrary suffix on all Enums, that "Enum" would be more appropriate.

So, I think the most common pattern may currently be 
<ECProperty propertyName="Foo" typeName="Foo"
It is subtle, but the way that you recognize it as an enum is that the typeName is anything other than a standard datatype name.

Reply:
we decided for no suffix on enums in our discussion. renamed to Portdirection